crates/re_types_builder/src/codegen/python.rs

```diff
@@ -513,7 +689,11 @@ fn code_for_struct(
             code.push_text("__repr__ = Archetype.__repr__", 1, 4);
         }
 
-        code.push_text(quote_array_method_from_obj(overrides, objects, obj), 1, 4);
+        code.push_text(
+            quote_array_method_from_obj(ext_class, overrides, objects, obj),
+            1,
+            4,
```

Comment:
off-topic, but I find the `1, 4,` really confusing and hard to read.

To me it would be more natural with the argument-order `indentation, text, newlines`, since that is the order they will be emitted in the code.

Reply:
Yeah, after working in this code I want to build a clearer helper function / macro.